tools/lint.sh

```diff
@@ -4,9 +4,32 @@
 
 set -e  # exit on error
 
-if [[ ! -z "$1" ]]; then
-    cd "$1"
-fi
```

Comment:
I didn't see any place that the changing of directory was being used so omitted it from the command line parsing.